WalletWasabi/Tor/TorSettings.cs

```diff
@@ -17,34 +18,18 @@ public class TorSettings
 		/// <param name="distributionFolderPath">Full path to folder containing Tor installation files.</param>
 		public TorSettings(string dataDir, string logFilePath, string distributionFolderPath)
 		{
-			TorDir = Path.Combine(dataDir, "tor");
-			TorBinaryDir = Path.Combine(TorDir, "Tor");
+			var torBinary = MicroserviceHelpers.GetBinaryPath(Path.Combine("Tor", "tor"));
```

Comment:
This should be a parameter of `TorSettings` too. `TorSettings` was meant to be purely data wrapper class that does not "find paths to something".